ci/deploy.ps1

```diff
@@ -0,0 +1,39 @@
+. "$PSScriptRoot\common.ps1"
+
+Start-Event "deploy-game" "build-gdk-third-person-shooter-:windows:"
+pushd "spatial"
+    # Get the short commit hash of this build
+    $commit_id = (get-item env:BUILDKITE_COMMIT).Value.Substring(0, 7)
```

Comment:
The shooter game project is unlikely to change regularly, perhaps we should use the GDK version here instead?

Reply:
Raised https://improbableio.atlassian.net/browse/UNR-1126 to look in to this